docs/source/frequently_asked_questions.ipynb

```diff
@@ -1558,6 +1558,57 @@
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How do I specify primitives on a per column or per entity basis?\n",
+    "\n",
+    "When using DFS to generate features, you may wish to use only certain features or entities for specific primitives. This can be done through the `primitive_options` parameter. The `primitive_options` parameter ",
+    "is a dictionary or list of dictionaries that maps a primitive or a tuple of primitives to a dictionary containing options for the primitive(s). This parameter can also be a list of option dictionaries if the primitive \n",
+    "takes multiple inputs. Each dictionary supplies options for their respective input column. There are multiple ways to control how primitives get applied through these options:\n",
+    "\n",
+    "- Use `ignore_entities` to specify entities that should not be used to create features for that primitive. It is a list of entity ids to ignore.\n",
+    "\n",
+    "- Use `include_entities` to specify the only entities to be included to create features for that primitive. It is a list of entity ids to include.\n",
+    "\n",
+    "- Use `ignore_variables` to specify variables in an entity that should not be used to create features for that primitive. It is a dictionary mapping an entity id to a list of variable names to ignore.\n",
+    "\n",
+    "- Use `include_variables` to specify the only variables in an entity that should be used to create features for that primitive. It is a dictionary mapping an entity id to a list of variable names to include.\n",
+    "\n",
+    "You can also use `primitive_options` to specify which entities or variables you wish to use as groupbys for groupby transformation primitives:\n",
+    "\n",
+    "- Use `ignore_groupby_entities` to specify entities that should not be used to get groupbys for that primitive. It is a list of entity ids to ignore.\n",
+    "\n",
+    "- Use `include_groupby_entities` to specify the only entities that should be used to get groupbys for that primitive. It is a list of entity ids to include.\n",
+    "\n",
+    "- Use `ignore_groupby_variables` to specify variables in an entity that should not be used as groupbys for that primitive. It is a dictionary mapping an entity id to a list of variable names to ignore.\n",
+    "\n",
+    "- Use `include_groupby_variables` to specify the only variables in an entity that should be used as groupbys for that primitive. It is a dictionary mapping an entity id to a list of variable names to include.\n",
+    "\n",
+    "Here is an example of using some of these options:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es = ft.demo.load_mock_customer(return_entityset=True)\n",
+    "\n",
+    "feature_matrix, feature_defs = ft.dfs(entityset=es,\n",
```

Comment:
i see this warning when I build the docs running this code. is it intentional? 

<img width="735" alt="Screen Shot 2019-09-26 at 1 29 53 PM" src="https://user-images.githubusercontent.com/371792/65710868-ce5a8880-e061-11e9-9da6-6cf56909e859.png">

Reply:
if it is intentional, any idea why it's not doing the string formating with the variable and entity?